cli/src/cmd/new.rs

```diff
@@ -161,12 +166,15 @@ fn build_sh_contents(name: &str) -> String {
 
 PROJNAME={}
 
-CARGO_INCREMENTAL=0 &&
-cargo build --release --features generate-api-description --target=wasm32-unknown-unknown --verbose &&
-wasm2wat -o target/$PROJNAME.wat target/wasm32-unknown-unknown/release/$PROJNAME.wasm &&
-cat target/$PROJNAME.wat | sed "s/(import \"env\" \"memory\" (memory (;0;) 2))/(import \"env\" \"memory\" (memory (;0;) 2 16))/" > target/$PROJNAME-fixed.wat &&
-wat2wasm -o target/$PROJNAME.wasm target/$PROJNAME-fixed.wat &&
-wasm-opt -Oz target/$PROJNAME.wasm -o target/$PROJNAME-opt.wasm"##,
+# cargo clean
+# rm Cargo.lock
+
+CARGO_INCREMENTAL=0 cargo build --release --features generate-api-description --target=wasm32-unknown-unknown --verbose
```

Comment:
Why have you removed `&&` here?
I think our approach was to add `set -e`, wasn't it?

Reply:
I copied what was in the build.sh script from the examples. Can you `suggest` an improvement to this?

Reply:
Just add `set -e` on top of the script for now